installer/datafiles/ruby.data

```diff
@@ -3,38 +3,14 @@ RUBY_DEST:                '/opt/microsoft/omsagent/ruby'
 
 %Files
```

Comment:
The file you got by running your private version of genRubyInstaller has no changes whatsoever compared to the base line. Those changes are purely content re-formatting related. In fact running the genRubyInstaller does not give anything as compared to the new changes for ruby. Because even if the new gem is installed from the intermediate or build location , the gem gets binplaced under /opt/Microsoft path by default. I am going to document what are my findings and how I got the new changes of ruby.data. Please bare with me for the new code review  specifically for ruby installer related fixes.